package Entities.AlgoGen;

message Address {
   optional int32 IdAddress = 1 [default = 0];
   optional double Lat = 2 [default = 0];
   optional double Lon = 3 [default = 0];
}

message List_Address {
   repeated Address items = 1;
}
